perception/shape_estimation/include/shape_estimation/corrector/bicycle_corrector.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 Autoware Foundation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SHAPE_ESTIMATION__CORRECTOR__BICYCLE_CORRECTOR_HPP_
+#define SHAPE_ESTIMATION__CORRECTOR__BICYCLE_CORRECTOR_HPP_
+
+#include "shape_estimation/corrector/vehicle_corrector.hpp"
+#include "utils.hpp"
+
+class BicycleCorrector : public VehicleCorrector
+{
+public:
+  explicit BicycleCorrector(const bool use_reference_yaw = false)
+  : VehicleCorrector(use_reference_yaw)
+  {
+    corrector_utils::CorrectionBBParameters params;
+    params.min_width = 0.3;
+    params.max_width = 1.2;
+    params.default_width = (params.min_width + params.max_width) * 0.5;
+    params.min_length = 1.0;
+    params.max_length = 2.6;
```

Comment:
Can you tell me how did you decide this max length?
I searched and found longer motorcycle with 2.65m from [HONDA](https://www.honda.co.jp/VT1300/VT1300CR/size/).

How about expanding this a little bit?
```suggestion
    params.max_length = 2.8;
```

Reply:
Thank you so much @YoshiRi -san, some BossHoss models are around 2.7 m, and tandem bikes could be bigger but I don't have a reference. 

We can take your link as a reference.